gnomad/utils/transcript_annotation.py

```diff
@@ -0,0 +1,59 @@
+"""Utils module containing generic functions that are useful for adding transcript expression-aware annotations."""
+from typing import Callable, Optional
+
+import hail as hl
+
+
+def summarize_rsem_mt(
+    rsem_mt: hl.MatrixTable,
+    rsem_expr: hl.expr.NumericExpression,
+    tissue_expr: hl.expr.StringExpression,
+    summary_agg_func: Optional[Callable] = None,
+    tissue_as_row: bool = False,
```

Comment:
While I'm writing next steps, I found that we'll have to write code every time for the case 'tissue_as_row'. Is it really useful?

Reply:
see change. I think we should only store as HT not in array, then use the other function when useful